src/server/game/Entities/Unit/Unit.h

```diff
@@ -2030,10 +2030,12 @@ class TC_GAME_API Unit : public WorldObject
         void _RegisterDynObject(DynamicObject* dynObj);
         void _UnregisterDynObject(DynamicObject* dynObj);
         DynamicObject* GetDynObject(uint32 spellId);
+        std::list<DynamicObject*> GetDynObjects(uint32 spellId);
```

Comment:
Chose list because the container used to store the objects, are lists.
What is the advantage of returning a different container in this case?

```
typedef std::list<DynamicObject*> DynObjectList;
DynObjectList m_dynObj;

typedef std::list<GameObject*> GameObjectList;
GameObjectList m_gameObj;
```